pheme/templatetags/charts/__init__.py

```diff
@@ -66,19 +71,19 @@ def _build_legend(
     """
     legend_elements = ""
     x_pos = 0
+    y_pos = 0
+    text_x = font_size + 2
     for label, color in label_color.items():
-        text_x = x_pos + font_size + 2
         legend_elements += __LEGEND_ELEMENT.format(
             x=x_pos,
+            y=y_pos,
             font_size=font_size,
             font_family=font_family,
             color=color,
-            half_font_size=font_size / 2,
+            text_y=y_pos + 5,
             label=label,
             text_x=text_x,
```

Comment:
```suggestion
            text_x=text_x_offset,
```

Sorry I needed to think about this, to get what this variable is for ...